app/views/handlers/feedback.py

```diff
@@ -114,6 +61,125 @@ def handle_post(self):
         self._session_store.session_data.feedback_count += 1
         self._session_store.save()
 
+    def get_question_schema(self):
+        if self._schema.form_type == "C":
```

Comment:
There is quite a bit of duplication here, I would be more incline to do this 

```
detail_answers_option_map = {
            "C": ["General", "This establishment", "People who live here"],
            "I": ["General", "Accommodation", "Personal details"],
            "H": ["General", "People who live here", "Visitors"],
            }

        options = ({"label": lazy_gettext(value), "value": value} for value in detail_answers_option_map.get(self._schema.form_type or "H"))

```
Not sure about the default to "H" which both this and yours does, might want to check what we want to do if there is no form_type

Reply:
Refactored, I'll ask about the default option, thanks!👍